File: 1651-shuffle-string/solution.cpp
class Solution {
public:
    string restoreString(string s, vector<int>& indices) {
        
        for(int i=0;i<indices.size();++i){
            for(int j=i;j<indices.size();++j){
                if(indices[i] > indices[j]){
                    swap(indices[i],indices[j]);
                    swap(s[i],s[j]);
                }
            }
        }
        return s;
    }
};

File: 1651-shuffle-string/solution.py
class Solution:
    def restoreString(self, s: str, indices: List[int]) -> str:
        map = {}

        for i in range(0, len(indices)):
            map[indices[i]] = s[i]
    
        ls = sorted(map)
    
        ans = ""
        for i in ls:
            ans += map[i]

        return ans
